ament_pclint/ament_pclint/main.py

```diff
@@ -0,0 +1,372 @@
+#!/usr/bin/env python3
+
+# Copyright 2017 Apex.AI, Inc.
+#
+#
+# This file contains modified code from the following open source projects
+# published under the licenses listed below:
+#
+#
+# # Copyright 2014-2015 Open Source Robotics Foundation, Inc.
+# #
+# # Licensed under the Apache License, Version 2.0 (the "License");
+# # you may not use this file except in compliance with the License.
+# # You may obtain a copy of the License at
+# #
+# #     http://www.apache.org/licenses/LICENSE-2.0
+# #
+# # Unless required by applicable law or agreed to in writing, software
+# # distributed under the License is distributed on an "AS IS" BASIS,
+# # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# # See the License for the specific language governing permissions and
+# # limitations under the License.
+
+import argparse
+import concurrent.futures
+import multiprocessing
+import os
+from shutil import which
+import subprocess
+import sys
+import time
+from xml.etree import ElementTree
+from xml.sax.saxutils import escape
+from xml.sax.saxutils import quoteattr
+
+
+def main(argv=sys.argv[1:]):
+    extensions = ['c', 'cc', 'cpp', 'cxx', 'c++']
```

Comment:
This list doesn't seem to match the one in the CMake package?

Reply:
Do you mean the list in `ament_cmake_pclint_lint_hook.cmake` (https://github.com/ament/ament_lint/pull/98/files#diff-ebf9131ad30342b5b69552b98477905a)? You are right, it's missing the `.c++` extension.